de.wwwtech.ColorMate.json

```diff
@@ -0,0 +1,48 @@
+{
+  "app-id": "de.wwwtech.ColorMate",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.36",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": ["org.freedesktop.Sdk.Extension.rust-stable"],
+  "command": "colormate",
+  "finish-args": [
+    "--share=network",
```

Comment:
What does it needs network access for?

Reply:
Dunno, this has been generated by GNOME Builder. I guess it doesn't?

Reply:
Yeah, that's just the default template